core/src/mender-artifact.c

```diff
@@ -1035,6 +1035,18 @@ artifact_read_meta_data(mender_artifact_ctx_t *ctx) {
     return MENDER_DONE;
 }
 
+/**
+ * @brief Callback function to be invoked to perform the treatment of the data from the artifact
+ * @param type Type from header-info payloads
+ * @param meta_data Meta-data from header tarball
+ * @param filename Artifact filename
+ * @param size Artifact file size
+ * @param data Artifact data
+ * @param index Artifact data index
+ * @param length Artifact data length
+ * @param dl_data Download data for the artifact
```

Comment:
This is not the "Artifact" but the payload, right?

Reply:
Not even the payload, but one file inside the payload :thinking:

Reply:
It's "artifact download data", so... :man_shrugging:

Reply:
Or meta-data, actually. But I called the variable `dl_data` everywhere and the type is also called like that, so I'll stick to it. At least for now.